mern-ecommerce/routes/order.js

```diff
@@ -29,15 +29,23 @@ router.get(
   requireSignin,
   isAuth,
   isAdmin,
-  getStatusValues
+  getStatus
 );
 
 router.put(
   '/order/:orderId/status/:userId',
   requireSignin,
   isAuth,
   isAdmin,
-  updateOrderStatus
+  updateOrder
+);
+
+router.delete(
+  '/order/:orderId/:userId',
+  requireSignin,
+  isAuth,
+  isAdmin,
+  remove
 );
```

Comment:
There is no code snippet provided. Please provide the code snippet you want me to review.